analysis/report_templates/default.html

```diff
@@ -35,10 +35,15 @@
         <h1 class="center-align brown-text text-darken-3">
             FuzzBench: {{ experiment.name }} report
         </h1>
+        {% if git_hash %}
+        <h5 class="center-align brown-text text-darken-3">
+            Commit hash: <a href="https://github.com/google/fuzzbench/commits/{{ git_hash }}">{{ git_hash }}</a>
+        </h5>
+        {% endif %}
```

Comment:
As this is not the most important information for most users, consider putting this at the bottom of the report with small letters.

Reply:
Done, moved to experiment data section.